README.md

```diff
@@ -0,0 +1,41 @@
+# gRPC-rs
+
+[![Build Status](https://travis-ci.org/pingcap/grpc-rs.svg)](https://travis-ci.org/pingcap/grpc-rs)
+
+The Rust wrapper of [gRPC Core](https://github.com/grpc/grpc). [Grpc](http://www.grpc.io) is a high performance, open source, general RPC framework that puts mobile and HTTP/2 first.
+
+This project is still under developement, not all features are supported.
```

Comment:
-> This project is still under development, not all features are supported. See [Status](#status) for more information.